cs16-wiki/src/components/ConsoleSection.tsx

```
@@ -0,0 +1,45 @@
+      className="mb-8"
+      initial={{ opacity: 0, y: 20 }}
+      animate={{ opacity: 1, y: 0 }}
+      transition={{ duration: 0.4 }}
+    >
+      {/* Section Header */}
+      <div className={`
+        flex items-center gap-2 mb-4 pb-2 border-b
+        ${isTechnical ? 'border-cs-yellow/30' : 'border-cs-green/30'}
+      `}>
+        <span className={`text-xs ${isTechnical ? 'text-cs-yellow' : 'text-cs-green'}`}>
+          //
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)